### DISCUSSION TITLE
Change 〉sign to > for cmd on Windows 10

### DISCUSSION THREAD

**Discussion by BerezkovN** [nushell/nushell#6930] [Q&A] [answered]
That question mark sign is annoying.

![image](https://user-images.githubusercontent.com/45544538/198689321-81fc9621-bdf2-4758-a6a3-d6851f11cbfa.png)

**Comment by fdncred** [comment, marked as answer]
You can use a nerdfont to fix this. If you don't want to change fonts then you can set the [prompt indicator](https://www.nushell.sh/book/3rdpartyprompts.html#oh-my-posh) in the config file.
